### DISCUSSION TITLE
RHS History (or Threads/Search history)

### DISCUSSION THREAD

**Discussion by jespino** [mattermost/community#138] [Enhancements]
Have the history of things that you have open in the RHS to be able to go back to there. Search results, open thread, mentions. Some of them are going to be more useful than others, but I always miss that "back to the previous open thread", specially with team switches.

**Comment by chetanyakan** [comment]
Maybe we can leverage the browser back button to implement this.

**Reply by esethna** [reply]
Ya that's an interesting thought, since right now the RHS content isn't in the stack of browser history

**Comment by esethna** [comment]
We had a design ticket for a "recent" element in the RHS that would do something similar: https://mattermost.atlassian.net/browse/MM-24111

Especially with CRT I think this becomes more valuable
